src/System.Private.CoreLib/shared/System/Diagnostics/StackTrace.cs

```diff
@@ -339,7 +339,11 @@ internal string ToString(TraceFormat traceFormat)
         private static bool ShowInStackTrace(MethodBase mb)
         {
             Debug.Assert(mb != null);
-            return !(mb.IsDefined(typeof(StackTraceHiddenAttribute)) || (mb.DeclaringType?.IsDefined(typeof(StackTraceHiddenAttribute)) ?? false));
+
+            // Don't show AggressiveInlining or StackTraceHidden items
```

Comment:
I find this expression pretty hard to read. Consider breaking it out?

Reply:
Broke it out.